examples/user_guide/Glasbey_Colors.ipynb

```diff
@@ -0,0 +1,133 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "import colorcet as cc\n",
+    "import holoviews as hv\n",
+    "\n",
+    "hv.extension('bokeh')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll use bokeh for displaying these colormaps so that you can zoom in for a closer look."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "xs, _ = np.meshgrid(np.linspace(0, 1, 256), np.linspace(0, 1, 10))\n",
+    "\n",
+    "def colormap(name, cmap=None, bounds=(0, 0, 256, 1), array=xs, **kwargs):\n",
+    "    if cmap is None:\n",
+    "        cmap = cc.palette[name]\n",
+    "    options = hv.opts.Image(xaxis=None, yaxis=None, width=900, height=100, cmap=cmap, toolbar='above', **kwargs)\n",
+    "    return hv.Image(array, bounds=bounds, group=name).opts(options)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The code requires some starting set of colors, which by default is just white. Using the default, the second color is always very dark, which doesn't make for a very recognizable first color to choose for plotting. By adding both white and black as the starting colors, the initial colors in the subsequent sequence seem more useful, in that they are actually nameable colors. All glasbey cmaps included in colorcet are initialized with black and white. In addition - some also include the named categorical colormaps such as Category10 and Category20 used in [bokeh](https://bokeh.pydata.org/en/latest/docs/reference/palettes.html#bokeh.palettes.d3) and originally from [D3](https://github.com/d3/d3-3.x-api-reference/blob/master/Ordinal-Scales.md#categorical-colors). The naming of these colormaps follows a different pattern than that of continuous colormaps:\n",
+    "       \n",
+    "       glasbey_<starting_palette>[_no_black<if_no_black>]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hv.Layout([colormap(name) for name in sorted(list(cc.palette.keys())) if \"glasbey_\" in name]).cols(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In addition to these long form names, colorcet provides several aliases to the most useful of the glasbey colormaps. These include - `glasbey`, `Category10`, and `Category20`. `Category10` and `Category20` can be used as replacements for the bokeh colormaps of the same name since they start with the same sequence."
```

Comment:
I have gotten feedback from colorcet users that using the same name as any common colormap from a different package causes confusion, so I think `Category10` should be maybe `gCategory10`, and similarly for `Category20`. Yes, these colormaps are in a different namespace, but people have still had trouble...